src/50_Samples_%26_Templates/Product_Listing.html

```diff
@@ -99,7 +99,7 @@ <h4 class="category"><a href="/samples_templates/product_listing/preview/">More<
     </a>
   </amp-carousel>
 
-  <h5>[[.Title]]</h5>
+  <h3>[[.Title]]</h3>
```

Comment:
This section and the one below is missing margins in demo.
![listing](https://cloud.githubusercontent.com/assets/820891/23922431/393c4ffe-08fa-11e7-968e-ae53e908706e.png)